org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/special/PackageReGenerator.java

```diff
@@ -204,7 +219,12 @@ public PackageReGenerator setNpmId(String npmId) {
   private Set<String> sourcePackages = new HashSet<>();
   private Map<String, TerminologyResourceEntry> entries = new HashMap<>();
   private Set<String> modeParams;
-  private List<CanonicalResource> resources = new ArrayList<CanonicalResource>();
+
+  public List<CanonicalResource> getResources() {
```

Comment:
Nitpick: could be a Lombok getter.

Just add `import lombok.Getter;` to the imports and annotate the `resources` field with `@Getter`. We should use these more to reduce clutter.

Reply:
Done